homeworks/lesia.turua_lesiaturuta/PokemonProject/scripts/pokemon.js

```diff
@@ -1,273 +1,21 @@
-            'fire',
-        ],
-    },
-    {
-        abilities: [
-            'Blaze',
-        ],
-        detailPageURL: '/us/pokedex/charmeleon',
-        weight: 41.9,
-        weakness: [
-            'Water',
-            'Ground',
-            'Rock',
-        ],
-        number: '0005',
-        height: 43.0,
-        collectibles_slug: 'charmeleon',
-        featured: 'true',
-        slug: 'charmeleon',
-        name: 'Charmeleon',
-        ThumbnailAltText: 'Charmeleon',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/005.png',
-        id: 5,
-        type: [
-            'fire',
-        ],
-    },
-    {
-        abilities: [
-            'Blaze',
-        ],
-        detailPageURL: '/us/pokedex/charizard',
-        weight: 9999.0,
-        weakness: [
-            'Water',
-            'Electric',
-            'Rock',
-        ],
-        number: '0006',
-        height: 1102.0,
-        collectibles_slug: 'charizard',
-        featured: 'true',
-        slug: 'charizard',
-        name: 'Charizard',
-        ThumbnailAltText: 'Charizard',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
-        id: 6,
-        type: [
-            'fire',
-            'flying',
-        ],
-    },
-    {
-        abilities: [
-            'Drought',
-        ],
-        detailPageURL: '/us/pokedex/charizard',
-        weight: 221.6,
-        weakness: [
-            'Water',
-            'Electric',
-            'Rock',
-        ],
-        number: '0006',
-        height: 67.0,
-        collectibles_slug: 'charizard',
-        featured: 'true',
-        slug: 'charizard',
-        name: 'Charizard',
-        ThumbnailAltText: 'Charizard',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
-        id: 6,
-        type: [
-            'fire',
-            'flying',
-        ],
-    },
-    {
-        abilities: [
-            'Tough Claws',
-        ],
-        detailPageURL: '/us/pokedex/charizard',
-        weight: 243.6,
-        weakness: [
-            'Ground',
-            'Rock',
-            'Dragon',
-        ],
-        number: '0006',
-        height: 67.0,
-        collectibles_slug: 'charizard',
-        featured: 'true',
-        slug: 'charizard',
-        name: 'Charizard',
-        ThumbnailAltText: 'Charizard',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
-        id: 6,
-        type: [
-            'fire',
-            'dragon',
-        ],
-    },
-];
+let pokemons = [];
 
 const cardsEl = document.querySelector('[data-cards]');
 const selectSearchEl = document.querySelector('[data-filter-select-search]');
-const selectSearchRadioOnlyNewEl = document.querySelector('[data-filter-new]');
-const selectSearchRadioFavoritesEl = document.querySelector('[data-filter-favorites]');
+const selectSearchRadioHighEl = document.querySelector('[data-filter-high]');
```

Comment:
Why do you add `selectSerch` to the names of all these elements? 
`selectSearchRadioHighEl` is not a search
`selectSearchEl` is not a select
...

You can pick proper name, like e.g. `searchEl`or `searchFilterEl`, `highFilterEl`